x-pack/plugins/apm/public/components/app/service_overview/service_overview_table.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiBasicTable, EuiBasicTableProps } from '@elastic/eui';
+import React from 'react';
+import styled from 'styled-components';
+
+/**
+ * The height for a table on the overview page. Is the height of a 5-row basic
+ * table.
+ */
+const tableHeight = 298;
```

Comment:
This will cause problems as soon as Eui changes anything related to the height of the column (padding, margin, border etc), right. Would be great if we ca make it dynamic eg:
```suggestion
const tableHeight = 5 * EuiTable.rowHeight;
```

Probably not a variable that's exposed to but might be worth asking for.

Reply:
The row doesn't have a height set explicitly, the actual table height is calculated by:

s = height of the sparkline chart (the tallest element in the row) = 24px
c = the table cell padding = 8px * 2
b = the table row borders and padding = 2px * 2
n = the number of rows = 5
h = height of the heading = 34px
p = height of the pagination = 40px

So it's ((s + c + b) * 5) + h + p = 298px

If we or EUI were to change any of these it would just make it so there's a bit of extra whitespace at the bottom of the chart or the table, so I think it's safe to leave it for now.